cypress/e2e/tests/structureSurvey/noTrialSpecific/onboardPractice.cy.ts

```diff
@@ -0,0 +1,144 @@
+  loginWebAdmin,
+  logoutWebAdmin,
+  assignSurveyToPatient,
+  unassignSurveyToPatient,
+  calculatedTime,
+} from "../../../helpers/adminHelper/adminHelper";
+
+import {
+  verifySurveyCard,
+  ONBOARDING_PRACTICE_SURVEY_ID,
+  verifyOnboardingExperienceOptions,
+  verifyPleaseProvideFeedbackScreen,
+  verifyQuestionsNotBeenAnswered,
+  checkAlertMessage,
+  currentDate,
+  SCHEDULE_ONCE,
+  submitSurvey,
+  CONTAINED_PRIMARY_BUTTON,
+  ALERT_NOTIFICATION,
+  verifyInputMessageError,
+  OPTION_RADIO_EXCELLENT,
+  FEEDBACK_TEXT_INPUT,
+  QUESTIONS_TEXT_INPUT,
+  verifyOnboardPracticeConfirmation,
+} from "../../../helpers/surveyHelper";
+
+import {
+  loginWebPatient,
+  logoutWebPatient,
+} from "../../../helpers/loginHelper";
+
+import {
+  ONBOARDING_PRACTICE_SURVEY,
+  THIS_FIELD_CANNOT_BE_LEFT_BLANK,
+} from "../../../strings/surveyStrings";
+
+import { SURVEY_CARD_BUTTON } from "../../../helpers/homeHelper";
+import { NEXT } from "../../../helpers/generalStrings";
+
+describe("notTrialSpecific - Onboarding Practice Survey", () => {
+  beforeEach(() => {
+    clearCache();
+    loginWebAdmin(Cypress.env("emailAdmin"), Cypress.env("passwordAdmin"));
+    assignSurveyToPatient("Alan Patient 20", ONBOARDING_PRACTICE_SURVEY);
+    logoutWebAdmin();
+
+    clearCache();
+    loginWebPatient(
+      Cypress.env("emailWebPatient"),
+      Cypress.env("passwordWebPatient"),
+    );
+
+    cy.get(SURVEY_CARD_BUTTON("Surveys"), { timeout: 10000 })
+      .should("be.visible")
+      .click();
+  });
+
+  afterEach(() => {
+    logoutWebPatient();
+    clearCache();
+    loginWebAdmin(Cypress.env("emailAdmin"), Cypress.env("passwordAdmin"));
+    unassignSurveyToPatient("Alan Patient 20", ONBOARDING_PRACTICE_SURVEY);
+    logoutWebAdmin();
+  });
+
+  it("How has your onboarding experience been so far? should be required, #5783", () => {
+    cy.get(ONBOARDING_PRACTICE_SURVEY_ID).click();
+    cy.get(CONTAINED_PRIMARY_BUTTON).contains(NEXT).click();
+    cy.wait(2000);
+    checkAlertMessage("Please select one option");
+    cy.get(OPTION_RADIO_EXCELLENT).click();
+  });
+
+  it("Experience feedback should be required, #5785", () => {
+    cy.get(ONBOARDING_PRACTICE_SURVEY_ID).click();
+    cy.get(OPTION_RADIO_EXCELLENT).click();
+    cy.get(CONTAINED_PRIMARY_BUTTON).contains(NEXT).click();
+    cy.get(CONTAINED_PRIMARY_BUTTON).contains(NEXT).click();
+    verifyInputMessageError(true, THIS_FIELD_CANNOT_BE_LEFT_BLANK);
+  });
+
+  it("Patient should be able to answer Onboarding Practice Survey when patient added question, #6111, #5781, #5782, #5786, #5787, #5788", () => {
+    const feedbackTextInput = "nothing, the system is great";
+    const questionsTextInput = "How does faeth work?";
+    verifySurveyCard(
+      ONBOARDING_PRACTICE_SURVEY,
+      SCHEDULE_ONCE(currentDate, calculatedTime),
+    );
+
+    cy.get(ONBOARDING_PRACTICE_SURVEY_ID).click();
+
+    verifyOnboardingExperienceOptions("25", "25", ONBOARDING_PRACTICE_SURVEY);
```

Comment:
I just noticed that these values are always the same, would it make sense to send only one parameter?

Reply:
Very good tip, Stefany! I took advantage and made these changes to other functions, I didn't know that you could pass 1 parameter to more of one function.

Reply:
Yes! Maybe we could keep `percentValue` instead of `percentValue`. wdyt?